src/main/java/com/smartsheet/wiremock/extensions/ApiScenarioTransformer.java

```diff
@@ -158,10 +157,11 @@ static String forMessage(String message, Integer errorCode) {
 
 	private static class RequestDiff {
 		private static String getDiff(Request request, JsonNode scenario) {
-			return diffHeaders(request, scenario) + " " +
-					diffUrl(request, scenario) + " " +
-					diffQueryParams(request, scenario) + " " +
-					diffBody(request, scenario).trim();
+			return String.format("%s %s %s %s",
+					diffHeaders(request, scenario),
+					diffUrl(request, scenario),
+					diffQueryParams(request, scenario),
+					diffBody(request, scenario).trim());
```

Comment:
Should this .trim() be outside of the last paren? (as in, String.format(...).trim())

Reply:
good catch. I thought that was where i put it.